test/performance/odp_ipsecfwd.c

```diff
@@ -365,14 +365,37 @@ static void print_usage(void)
 	       "Simple IPsec performance tester. Forward and process plain and ipsec packets.\n"
 	       "\n"
 	       "Examples:\n"
-	       "    %s -i ens9f1 -s /etc/odp/sa.conf -f /etc/odp/fwd.conf\n"
+	       "    %s -i ens9f1 -C /etc/odp/ipsecfwd.conf\n"
```

Comment:
I think this commit ("adjust help ...") could be squashed with the previus one that changes condig scheme as I do not see a benefit in keeping the help text update separate.

Reply:
V3.